### PR TITLE
#252/kickerz/team logo

### PR DESCRIPTION
**Pull request by conradlempert**

### CODE REVIEW

File: app/views/teams/_form.html.erb [hunk 1]
```diff
@@ -27,6 +27,17 @@
     </div>
   </div>
 
+  <div class="form-group row">
+          <%= f.label :avatar, t('activerecord.models.image.one'), :class => 'col-form-label col-lg-2'%>
```
Comment (ErikBrendel) [inline app/views/teams/_form.html.erb]: When I recall correctly, we agreed to not use the rocket Syntax.

Comment (conradlempert) [reply app/views/teams/_form.html.erb]: in unserem code ist die rocket syntax halt 100 mal. Ich änder das jetzt mal in meinem code für den pr, aber wenn wir das generell ändern wollen, müssen wir das fett refactoren.

File: app/views/teams/show.html.erb [hunk 2]
```diff
@@ -14,6 +14,7 @@
 
 <div class="page-header">
   <h1><%= t '.title', :default => model_class.model_name.human.titleize %></h1>
+  <%= image_tag @team.avatar_url || image_path('missing_avatar.png'), id: "preview", style: "width:350px; height:350px"%>
```
Comment (usame-algan) [inline app/views/teams/show.html.erb]: Would it be better here to use `max-width` or `max-height` to preserve the aspect ratio of the image?

File: app/models/team.rb [hunk 3]
```diff
@@ -9,6 +9,7 @@
 #  description   :text
 #  kind_of_sport :string
 #  private       :boolean
+#  avatar_data   :text
```
Comment (usame-algan) [inline app/models/team.rb:12]: Is there a difference between `avatar_data` and `:avatar`?

Comment (conradlempert) [reply app/models/team.rb:12]: Yes

File: app/views/teams/show.html.erb (hunk 2 above)
Comment (conradlempert) [reply app/views/teams/show.html.erb]: The issue says: "The picture should be resized to squared form 350 x 350. " Sounds to me like it should be stretched to that form.

Comment (lheilmann) [reply app/views/teams/show.html.erb]: @svitlanakalancha What do you think about resizing the image? In my opinion stretching the image is not the best solution.

File: app/views/teams/_form.html.erb [hunk 4]
```diff
@@ -27,6 +27,17 @@
...
+  <div class="form-group row">
+          <%= f.label :avatar, t('activerecord.models.image.one'), class: 'col-form-label col-lg-2'%>
+          <div class="col-lg-10">
+            <%= image_tag @team.avatar_url || image_path('missing_avatar.png'), id: "preview", style: "max-width:100px; max-height:100px"%>
+            <%= f.file_field :avatar, class: "form-control-file" %>
+            <% if @team.avatar %>
+                  <%= t('helpers.submit.delete', model: t('activerecord.models.image.one'))+ "  " %><%= f.check_box :remove_avatar %>
+              <% end %>
+    </div>
+    </div>
```
Comment (lheilmann) [inline app/views/teams/_form.html.erb]: Please double-check html structure.

- Label "Bild" is not formatted correctly
- Checkbox is not visible

screen shot 2018-01-15 at 14 49 53

File: db/migrate/20171215131703_add_avatar_data_to_teams.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class AddAvatarDataToTeams < ActiveRecord::Migration[5.1]
+  def change
+    add_column :teams, :avatar_data, :text
+  end
+end
```
Comment (lheilmann) [inline db/migrate/20171215131703_add_avatar_data_to_teams.rb:5]: Great!

**Review by lheilmann — CHANGES_REQUESTED** [review]
Some minor issues mainly in views. Please double-check and talk to PO before merging is possible. Rest looks good to me! :+1:

**Review by lheilmann — APPROVED** [review]

**Pull request merged**